examples/platform/silabs/MatterConfig.cpp

```diff
@@ -46,10 +46,12 @@
 #endif
 
 // TODO: We shouldn't need any platform specific includes in this file
-#if defined(SLI_SI91X_MCU_INTERFACE) && SLI_SI91X_MCU_INTERFACE == 1
-#include <platform/silabs/SiWx917/SiWxPlatformInterface.h>
+#if ((defined(SLI_SI91X_MCU_INTERFACE) && SLI_SI91X_MCU_INTERFACE == 1) || defined(EXP_BOARD))
 #include <platform/silabs/wifi/wiseconnect-interface/WiseconnectWifiInterface.h>
-#endif // SLI_SI91X_MCU_INTERFACE
+#if !defined(EXP_BOARD)
+#include <platform/silabs/SiWx917/SiWxPlatformInterface.h>
+#endif //! defined(EXP_BOARD)
+#endif //((defined(SLI_SI91X_MCU_INTERFACE) && SLI_SI91X_MCU_INTERFACE == 1 ) || defined(EXP_BOARD))
```

Comment:
Please brake these define guards into two distinct blocks. Combining them make this hard to understand.

Reply:
Added as per your comment

Reply:
I didn't mean having the include twice. I was to seperate the includes 

```
#if ...
#include SiWxPlatformInterface
#endif

#if ...
WiseconnectWifiInterface
#endif ...
```